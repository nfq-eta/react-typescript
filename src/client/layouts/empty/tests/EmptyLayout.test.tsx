import * as React from 'react';
import { shallow } from 'enzyme';

import { EmptyLayout } from '../EmptyLayout';

describe('EmptyLayout.js', () => {
  it('renders without crashing', () => {
    shallow(<EmptyLayout />);
  });

  it('matches snapshot', () => {
    const wrapper = shallow(<EmptyLayout />);
    expect(wrapper).toMatchSnapshot();
  });
});
