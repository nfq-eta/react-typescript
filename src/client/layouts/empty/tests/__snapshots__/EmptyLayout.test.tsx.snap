// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EmptyLayout.js matches snapshot 1`] = `
<div>
  Component name: EmptyLayout
</div>
`;
